calc/src/move.ts

```diff
@@ -58,6 +63,56 @@ export class Move {
     let data: MoveData & {name: string} = extend(true, {name}, MOVES[gen][name], options.overrides);
 
     // If isZMove but there isn't a corresponding z-move, use the original move
+    if (options.useMax && 'maxPower' in data) {
+      const maxMoveName: string = getMaxMoveName(data.type, options.species, !!(data.category === 'Status'));
```

Comment:
by the way i dont actually know where options.species or options.item would get assigned so this does almost nothing right now